config/DatabaseSetup.sql

```diff
@@ -142,3 +142,5 @@ FOREIGN KEY (CartID) REFERENCES Cart(CartID) ON DELETE CASCADE,
 FOREIGN KEY (SizeMapID) REFERENCES SizeMap(SizeMapID) ON DELETE SET NULL,
 FOREIGN KEY (RunID) REFERENCES Runs(RunID) ON DELETE SET NULL
 );
+
+CREATE TABLE Location(location varchar(255), PRIMARY KEY(location));
```

Comment:
Oh. I didn't realize piles was already storing locations dynamically. I'll make John's change and forget about the separate table.